core/src/main/scala/org/bitcoins/core/script/constant/StackPushOperationFactory.scala

```diff
@@ -21,7 +21,7 @@ trait StackPushOperationFactory {
     * @return
     */
   private def operations =
```

Comment:
This should be made a `val` do we don't have to run the comptuation associated with building `operations` everytime we call it.